common/app/common/Logback/KinesisAdapter.scala

```diff
@@ -0,0 +1,49 @@
+                        package common.Logback
+
+import java.util.concurrent.ThreadPoolExecutor
+
+import akka.actor.ActorSystem
+import akka.dispatch.MessageDispatcher
+import akka.pattern.CircuitBreaker
+import ch.qos.logback.classic.spi.ILoggingEvent
+import com.amazonaws.ClientConfiguration
+import com.amazonaws.auth.AWSCredentialsProvider
+import com.amazonaws.retry.{PredefinedRetryPolicies, RetryPolicy}
+import com.amazonaws.services.kinesis.AmazonKinesisAsyncClient
+import com.gu.logback.appender.kinesis.KinesisAppender
+import scala.concurrent.duration._
+import scala.concurrent.Future
+
+// LogbackOperationsPool must be wired as a singleton
+class LogbackOperationsPool(val actorSystem: ActorSystem)  {
+  val logbackOperations: MessageDispatcher = actorSystem.dispatchers.lookup("akka.logback-operations")
+}
+
+class SafeBlockingKinesisAppender(logbackOperations: LogbackOperationsPool) extends KinesisAppender[ILoggingEvent] {
```

Comment:
maybe it's worth adding some docs on why this needs to exist? To save people having to refer back to this pr

Reply:
👍